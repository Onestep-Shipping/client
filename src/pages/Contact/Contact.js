import React from 'react';
import './Contact.css';

import Header from '../../components/Header/Header.js';

const Contact = () => {
  return (
    <div class="homepage-container">
      <Header />
      <div class="body-container2">
        Contact
      </div>
    </div>
  );
};

export default Contact;
