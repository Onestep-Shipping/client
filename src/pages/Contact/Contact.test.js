import Contact from './Contact';
import React from 'react';
import { shallow } from 'enzyme';

let wrapper = shallow(<Contact />);

describe('Contact', () => {
  it('should render the Contact Component correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});