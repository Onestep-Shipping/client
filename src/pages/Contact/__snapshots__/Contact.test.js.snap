// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render the Contact Component correctly 1`] = `
<div
  className="homepage-container"
>
  <Header />
  <div
    className="body-container2"
  >
    Contact
  </div>
</div>
`;
